### DISCUSSION TITLE
No such file or directory: 'install'

### DISCUSSION THREAD

**Discussion by marcofochesato** [symfony/symfony#45636] [Symfony Book: The Fast Track]
Dear all,
while doing 
`symfony new --version=6.0-1 --book guestbook`,
I got:
[WEB] Installing Node dependencies (might take some time)

00h00m00s 0/0: : ERROR: [Errno 2] No such file or directory: 'install'

How can I fix it?
Marco

**Comment by lyrixx** [comment]
Hello, 

Could you paste the full output of the command please?

On my computer, it installs things, then breaks because I don't have a postgres db.

**Reply by marcofochesato** [reply]
There is an:
'unable to get project root'  
in red...

Here below the code (I don't have postgres service installed):

https://controlc.com/cb5b860d

**Comment by jayeshnana** [comment]
Hi Marco,

This resolves the problem (Ubuntu 20.04 LTS):

sudo apt remove cmdtest
sudo apt remove yarn
sudo npm install -g yarn
